purchase_backorder/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 
 {
     'name': 'Purchase Backorder Report',
-    'version': '12.0.1.0.0',
+    'version': '12.0.1.1.0',
```

Comment:
```suggestion
    'version': '12.0.1.0.0',
```
Let's runbot to bump the version